paper.tex

```diff
@@ -55,15 +56,15 @@ \section{Introduction}
 RQ2: How can generated texts be manipulated to trick fake text generators?
 
 
-The report is structured as follows: After this introduction chapter 2 presents related work, especially text and image generating models as well as fake image and text detectors. Chapter 3 presents the methodology of our work followed by chapter 4 presenting the results of our experiments. Afterwards the results will be discussed in chapter 5 and chapter 6 will conclude this paper.
+The report is structured as follows: After this introduction chapter 2 presents related work, especially text and image generative models as well as fake image and text detectors. Chapter 3 presents the methodology of our work followed by chapter 4 presenting the results of our experiments. Afterwards the results will be discussed in chapter 5 and chapter 6 will conclude this paper.
 
 \section{Related work}
 
-\subsection{Generating models}
+\subsection{Generative models}
 
-\subsubsection{Image generating models}
+\subsubsection{Image generative models}
 
-\subsubsection{Text generating models}
+\subsubsection{Text generative models}
```

Comment:
To my understanding the feedback regarding the paragraphs is meant, that we should remove those headings and rather just separate the sections by paragraphs. So if you could remove them as well, I think all feedback would be incorporated.